### PR TITLE
T280061: Update eslint-config-wikimedia to 0.20

### PR DESCRIPTION
**Pull request by medied**
https://phabricator.wikimedia.org/T280061

### Objective

Updates linter to 0.20. Please review and test carefully, more specific comments in-line. You should `rm -rf node_modules` and `npm install` first

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ module.exports = {
     sourceType: 'module'
   },
   rules: {
-    semi: [2, "never"]
+    semi: [2, "never"],
+    'no-nonoctal-decimal-escape': 'off'
```
Comment (medied) [inline .eslintrc.js:21]: No idea how to address this error other than here, any thoughts?

File: src/index.js [hunk 2]
```diff
@@ -28,94 +28,96 @@ function init( {
...
+	const foundSelectorLinks = []
+	const foundDetectLinks = []
+
+	const showPopup = ( e, refresh = false ) => {
+		e.preventDefault()
+
+		const popupId = Date.now()
+		const { target } = refresh ? last : e
+		const title = refresh ? last.title : target.getAttribute( 'data-wp-title' ) || target.textContent
+		// eslint-disable-next-line no-shadow
```
Comment (medied) [inline src/index.js:46]: `lang` is technically declared already from method arguments. Let me know if we want to update the name of one or other, it'd likely have repercussions in other parts of the code

Comment (stephanebisson) [reply src/index.js:46]: We should fix that by renaming `lang` to something else so it doesn't shadow the `lang` argument from the outer function.

File: .eslintrc.js (hunk 1 above)
Comment (stephanebisson) [reply .eslintrc.js:21]: What is this rule?

File: src/index.js (hunk 2 above)
Comment (medied) [reply src/index.js]: How about `localLang` to parallel the `globalLang` we already have in there? See https://github.com/wikimedia/wikipedia-preview/pull/112/commits/b4fde023b472ec1d9522ee1887cd8f6799e27ae6

**Pull request merged**